### PR TITLE
Autowarmth text formatting

### PR DESCRIPTION
**Pull request by zwim**
Correct wordings to twilight, where dusk and dawn are meant.

Make translation strings easier to manage in the formatted overview (no spaces at the end of a string; use string.rep instead).

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/8262)

### CODE REVIEW

File: plugins/autowarmth.koplugin/main.lua [hunk 1]
```diff
@@ -793,22 +793,23 @@ function AutoWarmth:showTimesInfo(title, location, activator, request_easy)
...
+            info_line(_("Sunrise:"), times, 5) ..
+            "\n" ..
+            info_line(_("Solar noon:"), times, 6, request_easy) ..
+            "\n" ..
+            info_line(_("Sunset:"), times, 7) ..
             _("  Dusk\n") ..
-            info_line(_("    Civil:      "), times, 8) ..
-            info_line(_("    Nautical:   "), times, 9, request_easy) ..
-            info_line(_("    Astronomic: "), times, 10, request_easy) ..
+            info_line(_("    Civil:"), times, 8) ..
```
Comment (poire-z) [inline plugins/autowarmth.koplugin/main.lua:808]: Shouldn't the leading spaces be left out, as you did for "\n" ?

Comment (zwim) [reply plugins/autowarmth.koplugin/main.lua:808]: Yep, good point.

File: plugins/autowarmth.koplugin/main.lua [hunk 2]
```diff
@@ -793,22 +794,23 @@ function AutoWarmth:showTimesInfo(title, location, activator, request_easy)
...
+            info_line(4, _("Astronomic:"), times, 2, request_easy) ..
+            info_line(4, _("Nautical:"), times, 3, request_easy)..
+            info_line(4, _("Civil:"), times, 4) ..
+            "  " .. _("Dawn\n") ..
+            info_line(0, _("Sunrise:"), times, 5) ..
+            "\n" ..
+            info_line(0, _("Solar noon:"), times, 6, request_easy) ..
+            "\n" ..
+            info_line(0, _("Sunset:"), times, 7) ..
+            "  " .. _("Dusk\n") ..
```
Comment (Frenzie) [inline plugins/autowarmth.koplugin/main.lua:808]: Better to put that \n outside.

File: plugins/autowarmth.koplugin/main.lua [hunk 3]
```diff
@@ -793,22 +794,23 @@ function AutoWarmth:showTimesInfo(title, location, activator, request_easy)
...
-
-            info_line(_("\nSunset:         "), times, 7) ..
-            _("  Dusk\n") ..
-            info_line(_("    Civil:      "), times, 8) ..
-            info_line(_("    Nautical:   "), times, 9, request_easy) ..
-            info_line(_("    Astronomic: "), times, 10, request_easy) ..
-            _("  Dusk\n") ..
-            info_line(_("Solar midnight: "), times, midnight_index, request_easy)
+            info_line(0, _("Solar midnight:"), times, 1, request_easy) ..
+            "  " .. _("Dawn\n") ..
```
Comment (Frenzie) [inline plugins/autowarmth.koplugin/main.lua:798]: Here too.

File: plugins/autowarmth.koplugin/main.lua [hunk 4]
```diff
@@ -793,22 +794,23 @@ function AutoWarmth:showTimesInfo(title, location, activator, request_easy)
...
+            info_line(0, _("Sunrise:"), times, 5) ..
+            "\n" ..
+            info_line(0, _("Solar noon:"), times, 6, request_easy) ..
+            "\n" ..
+            info_line(0, _("Sunset:"), times, 7) ..
+            "  " .. _("Dusk\n") ..
+            info_line(4, _("Civil:"), times, 8) ..
+            info_line(4, _("Nautical:"), times, 9, request_easy) ..
+            info_line(4, _("Astronomic:"), times, 10, request_easy) ..
+            "  " .. _("Dusk\n") ..
```
Comment (Frenzie) [inline plugins/autowarmth.koplugin/main.lua:812]: And here.

File: plugins/autowarmth.koplugin/main.lua (hunk 2 above)
Comment (zwim) [reply plugins/autowarmth.koplugin/main.lua:808]: +1 to all

File: plugins/autowarmth.koplugin/main.lua (hunk 3 above)
Comment (Frenzie) [reply plugins/autowarmth.koplugin/main.lua:798]:
```suggestion
            "  " .. _("Dawn") .. "\n" ..
```

File: plugins/autowarmth.koplugin/main.lua (hunk 2 above)
Comment (Frenzie) [reply plugins/autowarmth.koplugin/main.lua:808]:
```suggestion
            "  " .. _("Dusk") .. "\n" ..
```

File: plugins/autowarmth.koplugin/main.lua (hunk 4 above)
Comment (Frenzie) [reply plugins/autowarmth.koplugin/main.lua:812]:
```suggestion
            "  " .. _("Dusk") .. "\n" ..
```

**Comment by Frenzie** [comment]
Merging now so that the situation on Weblate will be significantly better.

**Pull request merged**